pnc-rest/src/main/java/org/jboss/pnc/rest/endpoint/ProjectEndpoint.java

```diff
@@ -75,12 +75,12 @@ public Response update(@ApiParam(value = "Project id", required = true) @PathPar
         return Response.ok().build();
     }
 
-    @ApiOperation(value = "Gets all Projects of a Product Version")
+    /*@ApiOperation(value = "Gets all Projects of a Product Version")
```

Comment:
Yes, it purposely breaks that part of the UI because the direct relationship between product versions and projects doesn't make sense, IMO, and should be removed.
The UI will be updated in NCL-480